[REFACTOR] GlobalMemberAdvice DB Connection Leak 오류 수정

### 🧼 리팩토링 대상 설명

GlobalMemberAdvice가 모든 요청에서 매번 DB를 호출하면서 HikariCP 커넥션 풀을 소진하고, 반환되지 않은 Connection이 발생 → DB Connection Leak 발생 (Apparent connection leak detected)
```
Caused by: JDBCConnectionException: Unable to acquire JDBC Connection
HikariPool-1 - Connection is not available, request timed out 
(total=10, active=10, idle=0)

Connection leak detection triggered
java.lang.Exception: Apparent connection leak detected
```

### ✅ 작업 항목

- [x] GlobalMemberAdvice.java 코드 수정

### 💬 비고

_No response_

--- a/src/main/java/gamo/web/home/controller/GlobalMemberAdvice.java
+++ b/src/main/java/gamo/web/home/controller/GlobalMemberAdvice.java
@@ -1,6 +1,7 @@
 package gamo.web.home.controller;
 
 import gamo.web.auth.UserPrincipal;
+import gamo.web.member.domain.Member;
 import gamo.web.member.dto.LoginResponseDTO;
 import gamo.web.member.service.MemberService;
 import lombok.RequiredArgsConstructor;
@@ -9,15 +10,14 @@ import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ModelAttribute;
 
 @ControllerAdvice
-@RequiredArgsConstructor
 public class GlobalMemberAdvice {
-
-    private final MemberService memberService;
 
     @ModelAttribute("member")
     public LoginResponseDTO addLoggedInMember(@AuthenticationPrincipal UserPrincipal user) {
         if (user == null) return null;
-        Long memberId = memberService.getMemberId(user);
-        return memberService.getMyInfo(memberId);
+
+        // DB 호출 없이 UserPrincipal에 있는 Member 사용
+        Member member = user.getMember();
+        return new LoginResponseDTO(member);
     }
 }

